caikit/runtime/servicers/runtime_info_servicer.py

```diff
@@ -0,0 +1,79 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Have pylint ignore Class XXXX has no YYYY member so that we can use gRPC enums.
+# pylint: disable=E1101
+
+# Standard
+from typing import Dict
+import sys
+if sys.version_info < (3, 10):
+    import importlib_metadata as importlib_meta
+else:
+    import importlib.metadata as importlib_meta
```

Comment:
This is failing with error `ModuleNotFoundError: No module named 'importlib_metadata'` in tox but when I run it locally it works...
```
$ python3.9
Python 3.9.16 (main, Dec  7 2022, 10:15:43) 
[Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import importlib_metadata
>>> importlib_metadata.packages_distributions()
{'tensorflow_text': ['tensorflow-text']
```

and should be compatible with py3.8 as seen from [here](https://github.com/python/importlib_metadata#compatibility)